source/js/Directions.js

```diff
@@ -7,55 +7,220 @@ class Directions extends HTMLElement {
+          padding: 3px;
+          overflow: hidden ;
+          background: #FFF6EC;
+        }
+
+        .listItemStyleShown {
+          border: 1px solid orange;
+          border-radius: 25px;
+          margin-left: auto;
+          margin-right: auto;
+          width: auto
+          height: auto;
+          padding: 3px;
+          background: #FFF6EC;
+        }
+    
+        .buttonstyle{
+          margin-left: 0.7em;
+          width: auto;
+          display: grid;
+          grid-template-columns: 10px auto auto
+          justify-content: flex-start;
+          align-items: top;
+          gap: 10px; 
+        }
+
+        .listItemStyle:hover {
+          cursor: pointer;
+          filter: brightness(96%);
+        }
+
+        .listItemStyleShown:hover {
+          cursor: pointer;
+          filter: brightness(96%);
+        }
+
+        .downArrow{
+          grid-column: 3;
+          
+          margin-right: 0.7em;
+          width: 1em;
+          height: 1em;
+        }
+        .upArrow{
+          grid-column:3;
+          margin-right: 0.7em;
+          width: 2em;
+          height: 2em;
+        }
+        
+        .text{
+          grid-column:2;
+          text-align: left;   
+          margin-block: 0 0;       
+        }
+
+        .olStyle{
+          margin-left: 0vw;
+          margin-right: 5vw;
+          display: flex;
+          flex-direction: column;
+          gap: 10px;
+        }
+    
+        .header{
+          font-style: italic;
+          font-size: 2em;
+          margin-left: auto;
+          margin-right: auto;
+          width: auto
+        }
 
+        .checkbox{
+          grid-column:1;
+          margin: 0;
         }
+
         `;
 
     const styleElem = document.createElement("style");
     styleElem.innerHTML = style;
 
     const directionList = searchForKey(data, "recipeInstructions");
 
-    const wrapper = document.createElement("article");
-    wrapper.classList.add("example");
+    const wrapper = document.createElement('article');
+    wrapper.classList.add('background');
+
 
     const directions = document.createElement("p");
     directions.classList.add("directions");
     directions.textContent = "Directions";
+    directions.classList.add("header");
     wrapper.appendChild(directions);
 
+    // let button_wrapper = document.createElement('button_wrapper');
+    // button_wrapper.classList.add('buttonstyle');
+
     // Parsing data to create the direction list.
-    const list = document.createElement("ol");
+    const list = document.createElement('ol');
+    list.classList.add("olStyle");
     for (let i = 0; i < directionList.length; i++) {
-      const listItem = document.createElement("li");
-      listItem.textContent = `${directionList[i].name}`;
-      list.appendChild(listItem);
+      let button = document.createElement('button');
+      let x = i+1;
+      let wrapper = document.createElement('div');
+      wrapper.classList.add('buttonstyle');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8460933)
```suggestion
      wrapper.classList.add("buttonstyle");
```